src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -201,14 +201,26 @@ public static bool IsWindowsConsumption(this IEnvironment environment)
         }
 
         /// <summary>
-        /// Gets a value indicating whether the application is running in a Windows or Linux Consumption (dynamic)
+        /// Gets a value indicating whether the application is running in one of our consumption SKUs (e.g.  Windows or Linux Consumption (Dynamic),
+        /// or Consumption V2).
         /// App Service environment.
         /// </summary>
         /// <param name="environment">The environment to verify.</param>
         /// <returns><see cref="true"/> if running in a Windows or Linux Consumption App Service app; otherwise, false.</returns>
         public static bool IsConsumptionSku(this IEnvironment environment)
         {
-            return IsWindowsConsumption(environment) || IsAnyLinuxConsumption(environment);
+            return IsWindowsConsumption(environment) || IsAnyLinuxConsumption(environment) || IsConsumptionV2Sku(environment);
```

Comment:
Update in the latest revision - I realized the this new consumption v2 Sku needs to be accounted for in our existing IsConsumptionSku helper.